docs/infra/Autorollers.md

```diff
@@ -1,66 +1,42 @@
+# Autorollers
+
 Several of our dependencies are automatically rolled (updated) by bots.
 
-## Clang to Engine
+<img src="https://media1.tenor.com/m/8WV-qfNTVRMAAAAd/autobots-rollout-cat.gif" height="200" />
+
+## Clang
 
-We use an auto-roller for Clang on [Linux](https://autoroll.skia.org/r/clang-linux-flutter-engine) and [macOS](https://autoroll.skia.org/r/clang-mac-flutter-engine) (Windows is pending availability of a Windows Clang package from the Fuchsia infra team). In case of build failures or other errors, ping the #hackers-engine channel on [Discord](../contributing/Chat.md).
+We use an auto-roller for Clang on [Linux](https://autoroll.skia.org/r/clang-linux-flutter-engine) and [macOS](https://autoroll.skia.org/r/clang-mac-flutter-engine) (Windows is pending availability of a Windows Clang package from the Fuchsia infra team). In case of build failures or other errors, ping the [`#hackers-engine channel`](https://discord.com/channels/608014603317936148/608021010377080866) on [Discord](../contributing/Chat.md).
```

Comment:
Actually; I don't know if there is auto rollers per os.

Reply:
Nice, fixed!